replica/database.cc

```diff
@@ -81,15 +81,7 @@ logging::logger dblog("database");
 // dirty_memory manager in that case.
 thread_local dirty_memory_manager default_dirty_memory_manager;
 
-namespace replica {
-
-inline
-flush_controller
-make_flush_controller(const db::config& cfg, backlog_controller::scheduling_group& sg, std::function<double()> fn) {
```

Comment:
nit: Likely the `make_flush_controller` move deserves its own patch to reduce the churn here

Reply:
Actually I moved `make_compaction_manager_config` out of the replica namespace, and didn't touch make_flush_controller (because the logic really belongs to compaction_manger).
I can do that in a preliminary patch so to reduce thew churn here.